icicle/backend/cpu/src/curve/cpu_msm.hpp

```diff
@@ -14,10 +14,16 @@
 #include "icicle/msm.h"
 #include "tasks_manager.h"
 #include "icicle/backend/msm_config.h"
+#ifdef LOG_UTILIZATION
+#include "icicle/utils/timer.hpp"
+#endif
 
 using namespace icicle;
 using namespace curve_config;
 
+#define LOG_EC_BATCH_SIZE 2
```

Comment:
is 2 the optimum?

Reply:
also the name is not that clear

Reply:
It was chosen as preparation for AVX but I can run a few checks to see what works better